src/nbl/ext/MitsubaLoader/CMitsubaMaterialCompilerFrontend.cpp

```diff
@@ -85,24 +86,24 @@ CMitsubaMaterialCompilerFrontend::EmitterNode* CMitsubaMaterialCompilerFrontend:
     if (_emitter->area.emissionProfile) {
         asset::material_compiler::IR::CEmitterNode::EmissionProfile profile;
         auto [image, sampler, meta] = getEmissionProfile(_emitter->area.emissionProfile);
-        if (!image) {
-            os::Printer::log("ERROR: Emission profile not loaded", ELL_ERROR);
-            return nullptr;
-        }
-        profile.texture = { image, sampler, 1.f };
-
-        auto inverseTransform = core::transpose(core::concatenateBFollowedByA(transform, _emitter->transform.matrix));
-        profile.right_hand = core::determinant(inverseTransform) >= 0.0f;
-        profile.up = core::normalize(inverseTransform[1]);
-        profile.view = core::normalize(inverseTransform[2]);
-
-        res->emissionProfile = profile;
-        float normalizeEnergy = _emitter->area.emissionProfile->normalizeEnergy;
-        if (normalizeEnergy < 0.0f) {
-            res->intensity *= meta->getMaxIntensity();
+        if (image) {
+            profile.texture = { image, sampler, 1.f };
+            auto inverseTransform = core::transpose(core::concatenateBFollowedByA(transform, _emitter->transform.matrix));
+            profile.right_hand = core::determinant(inverseTransform) >= 0.0f;
+            profile.up = core::normalize(inverseTransform[1]);
+            profile.view = core::normalize(inverseTransform[2]);
+
+            res->emissionProfile = profile;
+            float normalizeEnergy = _emitter->area.emissionProfile->normalizeEnergy;
+            if (normalizeEnergy < 0.0f) {
+                res->intensity *= meta->getMaxIntensity();
+            }
+            else if (normalizeEnergy > 0.0f) {
+                res->intensity *= normalizeEnergy / meta->getIntegral();
+            } 
```

Comment:
Still not according to : https://github.com/Devsh-Graphics-Programming/Nabla/pull/630#discussion_r1468267813